integrations/reporting_channels/gmail/gmail.py

```diff
@@ -1,10 +1,13 @@
 from __future__ import absolute_import
 import re
 import imaplib
+import logging
+from email.header import decode_header
```

Comment:
Why do we need this import?

Reply:
I was using logging as part of debugging, we can remove this imports